src/circuit/gadget/utilities/decompose_running_sum.rs

```diff
@@ -9,9 +9,11 @@
 //! The difference between each interstitial running sum output is constrained
 //! to be $K$ bits, i.e.
 //!                      `range_check`($k_i$, $2^K$),
-//! where `range_check(word, range)
-//!     = word * (1 - word) * (2 - word) * ... * ((range - 1) - word)`
-//! is an expression of degree range.
+//! where
+//! ```
```

Comment:
@daira Rust documentation is not plain Markdown; this gets interpreted as a Rust doctest. [We need an attribute here to mark it as not code](https://doc.rust-lang.org/rustdoc/documentation-tests.html#attributes):
```
```text
```